tests/foreman/cli/test_repository.py

```diff
@@ -2274,7 +2275,7 @@ def test_positive_sync(self):
         Repository.synchronize({'id': repo['id']})
         result = ssh.command(
             'ls /var/lib/pulp/published/yum/https/repos/{}/Library'
-            '/custom/{}/{}/ | grep .src.rpm'
+            '/custom/{}/{}/Packages/t/ | grep .src.rpm'
```

Comment:
is path always same on any system?

Reply:
yes ,, it would remain same.

Reply:
@ntkathole can you please approve and merge if fine with the above comment.